index.js

```diff
@@ -560,25 +604,33 @@ class ClientServiceBase {
    * @throws {ClientError}
    */
   _validateResponses(result1, result2) {
+    if (
+      (result1.getProofs().length === 0 || result2.getProofs().length === 0) &&
+      (result1.getProofs().length !== result2.getProofs().length)
+    ) {
+      return false;
+    }
+
```

Comment:
I guess this part is unnecessary? If there is no malicious behavior, the length should always be the same regardless of its number (0, 1, ...). And, we already have `result1.getProofs().length !== result2.getProofs().length` at the few lines down.

Reply:
> I guess this part is unnecessary? If there is no malicious behavior, the length should always be the same regardless of its number (0, 1, ...). And, we already have `result1.getProofs().length !== result2.getProofs().length` at the few lines down.

@jnmt 
Thank you Jun.

Sorry for confusing you.
I made this change for handling the case that Yamada-san mentioned earlier at
https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/57#discussion_r670114956

I removed the condition statement here
https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/57/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L539

The original behavior was if auditor has an empty proofs, then it will return true as consistent.
Maybe this code snippet should have an else block like
```
    if (
      (result1.getProofs().length === 0 || result2.getProofs().length === 0) &&
      (result1.getProofs().length !== result2.getProofs().length)
    ) {
      return false;
    } else {
      return true;
    }
```

Sorry for my misunderstanding.

Reply:
I think Jun's point is we can simply write as follows?
```
if (result1.getProofs().length !== result2.getProofs().length) {
    return false;
}
```

Reply:
> I think Jun's point is we can simply write as follows?

@feeblefakie @supl Almost yes. But we already have that kind of check [here](https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/57/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R618). That's why I'm thinking this part is unnecessary.

Reply:
@jnmt 
Jun, thank you for your explanation !

You're right.
This condition statement can check the case and return false.

We can remove this snippet
```
    if (
      (result1.getProofs().length === 0 || result2.getProofs().length === 0) &&
      (result1.getProofs().length !== result2.getProofs().length)
    ) {
      return false;
    }
```

I guess we still need to keep this snippet in case the proofs is empty but results are consistent, right?
https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/57/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L539

Reply:
@supl Sorry for the late. Yes, we need to keep your change on `_executeValidation()`. Thanks!

Reply:
Thank you.